src/main/java/mat/model/clause/CQLLibrary.java

```diff
@@ -332,6 +332,11 @@ public boolean isFhirMeasure() {
         return ModelTypeHelper.isFhir(getLibraryModelType());
     }
 
+    @Transient
```

Comment:
Hm.. Is it a setter or a getter? It uses is* notation but accepts a parameter... Confusing.